FPIS/Views/CreateProductForm.cs

```diff
@@ -0,0 +1,130 @@
+﻿using FPIS.Models;
+using FPIS.Views;
+using MaterialSkin.Controls;
+using System.Text.RegularExpressions;
+using static MaterialSkin.MaterialSkinManager;
+
+namespace FPIS.Views
+{
+    public partial class CreateProductForm : MaterialForm
+    {
+        bool _isDataValid = true;
+
+        public CreateProductForm()
+        {
+            InitializeComponent();
+            Theme.FormInstance = this;
+            Theme.Set(Themes.LIGHT);
+        }
+
+        private void ValidateProductName(string productName)
+        {
+            if (productName.Length == 0)
+            {
+                _isDataValid = false;
+                materialLabelProductNameError.Text = "Name is required!";
+            }
+        }
+
+        public static bool DoesProductExists(string productName, AppDbContext dbContext)
+        {
+            var product = dbContext.Products.FirstOrDefault(p => p.ProductName.ToLower() == productName.ToLower());
+
+            if (product != null)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        private void CreateProduct(string productName, AppDbContext dbContext)
+        {
+            try
+            {
+                var analysisItem = dbContext.AnalysisItems.Add(
+                        new()
+                        {
+                            ItemType = "Product"
+                        }
+                    ).Entity;
+
+                var newProduct = dbContext.Products.Add(
+                    new()
+                    {
+                        ProductName = productName
+                    }
+                 ).Entity;
+
+                dbContext.AnalysisProducts.Add(
+                        new()
+                        {
+                            ProductId = newProduct.Id,
+                            AnalysisItemId = analysisItem.Id
+                        }
+                    );
+
+                dbContext.SaveChanges();
+
+                MessageBox.Show(
+                    $"\"{productName}\" is successfully created.",
+                    "Success",
+                    MessageBoxButtons.OK,
+                    MessageBoxIcon.Information
+                    );

```

Comment:
Can you kindly clear the input field after the record is saved? To prevent any trips to the database when the user mistakenly clicks the button the second time.